### PR TITLE
500 internal server error page

### LINKED ISSUE #29: Design a nice "500 Internal Server Error" page for monolithic-layered app
Run MonolithicApplication and visit some page that generates server-side error.

### Actual Demo

image

### Expected Demo

image

(or something like this)

### Guide

1. Read https://www.baeldung.com/spring-boot-custom-error-page
2. Design 500.html page
3. Apply the 500 page to all resources: Drugs, Pharmacies, Diseases, etc.

### PR DESCRIPTION
**Pull request by youngInnovator**
Closes #29

### CODE REVIEW

**Review by naXa777 — CHANGES_REQUESTED** [review]
@youngInnovator,
1. add this magic text to your PR description: `Closes #29`.
3.  it's bad that this PR is adding a new feature and doesn't add new tests to verify that the feature works. Please write new tests for 500 page as a part of task #29.

**Comment by youngInnovator** [comment]
@naXa777 Suggested comments are addressed.

File: monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/web/MonolithicErrorControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.NOT_FOUND);
+        assertThat(response.getBody()).contains("<h4 class=\"alert-heading\"><i class=\"bi-exclamation-octagon-fill\"></i>Oops! Page not found</h4>");
+        assertThat(response.getBody()).contains("<p>Sorry, we couldn't find the page you were looking for.</p>");
+    }
+
+    @Test
+    @DisplayName("Test for internal server error page")
+    public void shouldReturnInternalServerErrorPage() {
+        String name = "Value longer than 255 characters cannot handled by database." +
```
Comment (naXa777) [inline monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/web/MonolithicErrorControllerTest.java]: Oh gosh! It seems like you discovered a new bug. Please create a ticket for it. Or I can do it myself, but later. Thank you!

**Review by naXa777 — APPROVED** [review]

File: monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/web/MonolithicErrorControllerTest.java (hunk 1 above)
Comment (youngInnovator) [reply monolithic/monolithic-layered/src/test/java/com/epam/training/microservices/monolithic/web/MonolithicErrorControllerTest.java]: Bug created #52. Test also updated so it don't need to be fixed after bug fixing.

**Comment by youngInnovator** [comment]
@aabarmin Kindly approve and merge the MR

**Review by aabarmin — APPROVED** [review]
Looks good to me

**Pull request merged**